### ISSUE TITLE
This should just use the eslint nodejs api, which colors the output and is much faster than calling child process (example code provided)

### ISSUE THREAD

**Issue by JohnnyFun** [leebeydoun/snowpack-eslint-plugin#3]
The eslint nodejs api already has a nice formatter that colors the console, so I made a plugin that just uses that directly. Much simpler code and much faster than calling node child process too.

Also, on a bigger project, this gets pretty slow, unless you pass through to eslint `--cache --cache-strategy content`, so I turned that on by default in my plugin too.

```
// https://www.snowpack.dev/reference/plugins
// https://eslint.org/docs/developer-guide/nodejs-api
const { ESLint } = require('eslint')

module.exports = function (snowpackConfig, pluginOptions) {
  return {
    name: 'snowpack-eslint',

    config() {
      this.pluginOptions = pluginOptions
      this.eslint = new ESLint({
        // Caching by default, otherwise large projects take forever. Caching by file content makes lint instant even on a large project.
        cache: true,
        // Default is 'metadata', but doesn't seem to un-cache when file changes (windows 10)
        cacheStrategy: 'content',
      })
    },

    run() {
      return this._runLint()
    },

    onChange() {
      return this._runLint()
    },

    async _runLint() {
      const patterns = this.pluginOptions.globs.split(' ')
      let results = await this.eslint.lintFiles(patterns)
      // TODO: consider auto-fixing certain errors (let to const, for instance)
      // 3. Modify the files with the fixed code.
      // await ESLint.outputFixes(results)
      const formatter = await this.eslint.loadFormatter('stylish')
      const resultText = formatter.format(results)
      console.log(resultText)
    }
  }
}
```

and in `snowpack.config.js`

```
['./snowpack.lint.js', {
  globs: 'src/**/**.js src/**/**.svelte', // or whatever you want...
}],
```

**Comment by leebeydoun** [comment]
This is a valid point, I'll pick this up sometime this week and publish and updated package.

**Comment by JohnnyFun** [comment]
Sounds good! No rush, of course. Just wanted to share my code, but didn't have enough time to clean it up enough for a PR.

**Comment by leebeydoun** [comment]
I have published a new version of the plugin at `v0.2.0`. It uses the `ESLint` Node API and provides a new interface that can be configured via the `snowpack.config.js` file. 

I've tested it briefly and it appears to work quite well. So I'll close this issue for now. If you do encounter any issues please let me know. 

Finally, thank you for taking the time to raise the issue and provide very useful feedback and advice.

**Comment by JohnnyFun** [comment]
Nice! Sure thing--thanks for integrating so quickly. The readme.md looks good to me too.
